main.py

```diff
@@ -282,14 +292,25 @@ def tmp(_, inp, out):
         round_zero=args.round_zero,
         global_ol_n_share=normal_outlier_count_global / w_count_global,
     )
+    if args.save_quantization:
+        torch.save(vars(args), args.save_quantization_pt + "/args.pt")
+        already_saved_weights = set()
+        for name, layer in nn.ModuleList(model.model.layers).named_modules():
```

Comment:
this line fails with Falcon(tested with 7B):
 ```
   for name, layer in nn.ModuleList(model.model.layers).named_modules():
  File "/home/optimus/conda/envs/py9/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1614, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'RWForCausalLM' object has no attribute 'model'
```

Reply:
use `get_layers(model)` from modelutils.py:
`for name, layer in get_layers(model)`

Reply:
Thank you for catching the error, I would fix it.